libs/api/repository/src/lib/gn4/elasticsearch/constant.ts

```diff
@@ -11,7 +11,14 @@ export const ES_SOURCE_SUMMARY = [
   'linkProtocol',
   'contactForResource.organisation',
   'contact.organisation',
+  'contact.email',
```

Comment:
This change is redundant with `FIELDS_SUMMARY`.
I think that `ES_SOURCE_SUMMARY` should be removed, what do you think @jahow .

@gkeimeHDF it's not your responsability, but I don't think that you need this change, as this constant is just used in the `related records` request.

Reply:
no big deal, there's probably some more cleanup to do but that doesn't have to be part of this PR; I'm fine either way